rosdep/base.yaml

```diff
@@ -887,6 +937,11 @@ gpsd:
   ubuntu: [gpsd]
 gradle:
   arch: [gradle]
+  debian:
+    buster: [gradle]
+    jessie: [gradle]
+    sid: [gradle]
```

Comment:
gradle appears to be available on all versions: https://packages.debian.org/stretch/gradle